web/models/donation_goal.ex

```diff
@@ -0,0 +1,33 @@
+defmodule CodeCorps.DonationGoal do
+  use CodeCorps.Web, :model
+
+  schema "donation_goals" do
+    field :title, :string
+    field :description, :string
+    field :amount, :integer # in cents
+    field :current, :boolean
```

Comment:
Hi Nikola, don't forget to add the relationship between project and donation goals(project has_many goals).